docs/examples/mne/load.md

```diff
@@ -0,0 +1,37 @@
+# Load Wearable Sensing Data
+
+```{admonition} Experiment
+:class: sidebar note
+The data were collected using a DSI-24 at a sampling rate of 300Hz. There were two conditions in which the user was asked to close and open their eyes.
+```
+
+## Example Data
+
+To follow along with this tutorial, you can download a sample data file from this [dropbox](https://www.dropbox.com/scl/fi/n1yhaco7fy397pu69l14u/Sample_DSI_24_Eyes_Closed.edf?rlkey=2vco7a100hhevg86i6vol1nmx&e=1&dl=0).
+
+## Import using MNE
+
+```{code-block} python
+:caption: Load Wearable Sensing EDF
+import mne
+
+# Define the file path of the EDF file on you computer
+edf_file_path = 'Sample_DSI_24_Eyes_Closed.edf'
```

Comment:
This should not have been committed. Perhaps your branch wasn't up to date with main? Please revert this as the file already exists elsewhere.